src/components/dialogs/PreferencesDialog/SpeedTabPanel/index.js

```diff
@@ -1,23 +1,32 @@
 import React, { Component} from 'react';
 import { inject, observer } from 'mobx-react';
-import lodash from 'lodash';
 
 import TextRow from '../fields/TextRow';
 import CheckRow from '../fields/CheckRow';
 import CheckValueRow from '../fields/CheckValueRow';
 import SelectRow from '../fields/SelectRow';
 
+const generateTimes = () => {
```

Comment:
Maybe it's interesting to extract so some utility module and add some tests to it. What do you think @Secretmapper ?